datashare-app/src/main/java/org/icij/datashare/com/mail/MailSender.java

```diff
@@ -68,11 +80,16 @@ private Message createMessage(Mail donneesEmail) throws MessagingException, Addr
         return message;
     }
 
-    private synchronized static Session getMailSession(String hostTransportMail, int portTransportMail) {
+    private synchronized static Session getMailSession(String hostTransportMail, int portTransportMail, String user, String password) {
```

Comment:
I did not know about the `synchronized` keyword. What is async here?

Reply:
it is in case it is used in a multithreaded context : this makes it thread safe.